src/pages/patientView/pathwayMapper/PatientViewPathwayMapper.tsx

```diff
@@ -201,46 +186,9 @@ export default class PatientViewPathwayMapper extends React.Component<
         );
     }
 
-    @computed get storeForAllData(): PatientViewPageStore | undefined {
+    @computed get patientStore(): PatientViewPageStore | undefined {
         let patientStore: PatientViewPageStore | undefined;
-        if (this.urlWrapperForAllGenes) {
-            return patientStore;
-        } else {
-            return undefined;
-        }
-    }
-
-    /**
-     * Here we clone the "query" field of the main store's URL Wrapper and enhance the cloned query
-     * with additional non-query genes. This new query object is used to fake a new URL Wrapper instance
-     * which is required to initialize a separate ResultsViewStore for the non-query genes.
-     */
-    @computed get urlWrapperForAllGenes(): PatientViewUrlWrapper | undefined {
-        let urlWrapper: PatientViewUrlWrapper | undefined;
-        if (
-            this.validNonQueryGenes.isComplete &&
-            this.validNonQueryGenes.result.length > 0
-        ) {
-            // fake the URL wrapper, we only need the query parameters with additional genes
-            const query: { [key: string]: any } = _.cloneDeep(
-                this.props.urlWrapper.query
-            );
-            query.gene_list = this.validNonQueryGenes.result.join(' ');
-
-            // we don't need a proper URL Wrapper here, just assign an object with a valid query field
-            urlWrapper = { query } as PatientViewUrlWrapper;
-        }
-
-        return urlWrapper;
-    }
-    @computed get validGenes() {
-        if (this.validNonQueryGenes.isComplete) {
-            // Valid genes are accumulated.
-            this.validNonQueryGenes.result.forEach(gene => {
-                this.accumulatedValidGenes[gene] = true;
-            });
-        }
-        return this.accumulatedValidGenes;
+        return patientStore;
```

Comment:
This is always undefined now. I don't think we need this ` @computed get patientStore()` function anymore.